data/857/991/99/85799199.geojson

```diff
@@ -60,6 +60,9 @@
     "name:spa_x_preferred":[
         "Bego\u00f1a"
     ],
+    "name:spa_x_preferred_x_variant":[
```

Comment:
Strange property name (preferred & variant), same as we saw in the `at` PR.

Reply:
I found the cause of this, and added an extra check to scrub all name pref/variant name props of the same language. Will add a new commit to fix.